client/pages/forgotPassword.jsx

```diff
@@ -0,0 +1,265 @@
+import React, { Component } from "react";
+import Link from "next/link";
+import {
+  getSecurityQuestionForUser,
+  submitSecurityQuestionAnswer,
+  resetPassword
+} from "../utils/apiWrapper";
+import {
+  Form,
+  Button,
+  FormGroup,
+  Label,
+  Input,
+  Card,
+  Alert,
+  CardBody,
+  CardTitle
+} from "reactstrap";
+import { setCookie } from "./../utils/cookie";
+import Router from "next/router";
+import { Head } from "../components";
+import "../public/styles/auth.scss";
+
+const EMAIL_REGEX =
+  "([a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)@([a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+).([a-zA-Z]{2,3}).?([a-zA-Z]{0,3})";
+// const PASSWORD_REGEX = "^(((?=.*[a-z])(?=.*[A-Z]))|((?=.*[a-z])(?=.*[0-9]))|((?=.*[A-Z])(?=.*[0-9])))(?=.{6,})";
+const SUCCESS = 200;
+class ForgotPasswordPage extends Component {
+  state = {
+    email: "",
+    question: "",
+    errorMessage: "",
+    answer: "",
+    pin: "",
+    password: "",
+    password2: "",
+    loadingAPI: false,
+    submitNewPassword: false
+  };
+
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleGetSecurityQuestion = async e => {
+    e.preventDefault();
+    console.log("HHERE");
```

Comment:
Don't forget to remove logs, there're some others as well

Reply:
yep! i think these were left in here from the h4i auth repo lol but i will remove all console logs